packages/extension-base/src/background/handlers/Extension.ts

```diff
@@ -476,7 +475,6 @@ export default class Extension {
     };
   }
 
-  // FIXME This looks very much like what we have in authorization
```

Comment:
fixme suggests refactor - I'm pretty content with how it looks currently, removing comment